CHANGELOG.md

```diff
@@ -4,11 +4,20 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 and this project adheres to [Calendar Versioning](https://calver.org/) of
 the following form: YYYY.0M.0D.
 
-## 2024.16.09
+## 2024.09.16
 
 ### Added
+
 ```
 - Designed a CarData component
 - Designed a Student Calander component
 - Designed a BankAccount component
 ```
+
+## 2024.10.03
```

Comment:
This is fine, but I tend to prefer the file to be organized in reverse chronological order. Most likely folks will want to see the latest changes first.